crepecake/src/test/java/com/google/cloud/tools/crepecake/image/UnwrittenLayerTest.java

```diff
@@ -52,6 +53,7 @@ public void setUpMocks() throws IOException {
     Mockito.when(mockBlobDescriptor.getDigest()).thenReturn(mockDiffId);
   }
 
+  // TODO: Test writeCompressedBlobTo
```

Comment:
do this or file it as a a bug

Reply:
This is no longer relevant since UnwrittenLayer only has an uncompressed blob. Removed